packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -287,6 +291,42 @@ export const samplerOperations = {
             },
         };
     },
+    getBancorSellQuotes(
+        makerToken: string,
+        takerToken: string,
+        takerFillAmounts: BigNumber[],
+        bancorService: BancorService,
+    ): SourceQuoteOperation<FillData, BancorQuoteData> {
+        return {
+            source: ERC20BridgeSource.Bancor,
+            encodeCall: _contract => {
+                return '0x';
+            },
+            handleCallResultsAsync: async (_contract, _callResults) => {
+                return Promise.all(
+                    takerFillAmounts.map(amt => bancorService.getQuoteAsync(makerToken, takerToken, amt)), // tslint:disable-line:promise-function-async
+                );
+            },
+        };
+    },
+    getBancorBuyQuotes(
+        makerToken: string,
+        takerToken: string,
+        makerFillAmounts: BigNumber[],
+        bancorService: BancorService,
+    ): SourceQuoteOperation<FillData, BancorQuoteData> {
+        return {
+            source: ERC20BridgeSource.Bancor,
+            encodeCall: _contract => {
+                return '0x';
+            },
+            handleCallResultsAsync: async (_contract, _callResults) => {
+                return Promise.all(
+                    makerFillAmounts.map(amt => bancorService.getQuoteAsync(takerToken, makerToken, amt)), // tslint:disable-line:promise-function-async
```

Comment:
Without a side this would be ambiguous and incorrect. 

Buying is `receive 100 of makerToken by selling X takerToken`. 
Selling is `receive X of makerToken by selling 100 takerToken`.

This is performing `receive X of takerToken by selling 100 makerToken`.

Reply:
Doesn't `getPathAndRate()` take a string for `amt`?

Reply:
Yeah let's just backlog buy support for bancor right now?

Reply:
made changes for `getSellQuote`, plz confirm sensical